network/transport/v2/transactionlist_handler.go

```diff
@@ -108,6 +110,8 @@ func (p *protocol) handleTransactionList(peer transport.Peer, envelope *Envelope
 
 	if msg.MessageNumber >= msg.TotalMessages {
 		p.cMan.done(cid)
+	} else {
+		p.cMan.resetTimeout(cid)
```

Comment:
According to the linked issue it should be reset upon receival, not after processing?

Reply:
From issue:

> Also reset the conversation timeout after each processed message.